src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith, SearchParams } from '../utils/searchHelper';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [currentPeopleList, setCurrentPeopleList] = useState<Person[]>(people);
```

Comment:
Initializing `currentPeopleList` with `people` directly might lead to unexpected behavior since `people` is initially an empty array. Consider initializing `currentPeopleList` as an empty array or updating it in the `useEffect` after `people` is fetched.